src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap

```diff
@@ -372,6 +405,13 @@ variables:
 [(14, 6), (14, 16)]   foo
 [(14, 23), (14, 31)]   z
 [(16, 7), (16, 15)]   prefName
+[(18, 27), (18, 39)]
+[(18, 41), (18, 52)]
+[(18, 54), (18, 69)]
+[(18, 28), (18, 29)]   c
+[(18, 34), (18, 38)]   c
+[(18, 43), (18, 50)]   frameId
+[(19, 10), (19, 15)]   v
```

Comment:
some of these vars are missing

```diff
+ function assignPatternFunc({c} = {c: 2}, { frameId }, [a, b] = [1, 2]) {
+     const v = c;
```

actually looks like a & b and maybe assignPatternFunc